games/Astrovoid.js

```diff
@@ -1,8 +1,11 @@
-// @title: Astrovoid
+/*
+@title: Astrovoid
```

Comment:
Shouldnt title & author be between quotation marks?

Reply:
no, not necessarily since it's regex, it isn't a typical string

Reply:
alright